main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gocarina/gocsv"
+	"github.com/gorilla/mux"
+)
+
+type Pokemon struct {
+	ID         string `csv:"ID"`
+	Name       string `csv:"Name"`
+	Type_1     string `csv:"Type_1"`
+	Type_2     string `csv:"Type_2"`
+	Total      int    `csv:"Total"`
+	HP         int    `csv:"HP"`
+	Attack     int    `csv:"Attack"`
+	Defense    int    `csv:"Defense"`
+	Sp_Atk     int    `csv:"Sp_Atk"`
+	Sp_Def     int    `csv:"Sp_Def"`
+	Speed      int    `csv:"Speed"`
+	Generation int    `csv:"Generation"`
+	Legendary  bool   `csv:"Legendary"`
+}
```

Comment:
Naming conventions say to use only `camelCase` or `PascalCase`. Please avoid the use of other case conventions (`snake_case`). These names are okay for the tags, but the variable name should be changed.

```suggestion
type Pokemon struct {
	ID         string `csv:"ID"`
	Name       string `csv:"Name"`
	Type1     string `csv:"Type_1"`
	Type2     string `csv:"Type_2"`
	Total      int    `csv:"Total"`
	HP         int    `csv:"HP"`
	Attack     int    `csv:"Attack"`
	Defense    int    `csv:"Defense"`
	SpAtk     int    `csv:"Sp_Atk"`
	SpDef     int    `csv:"Sp_Def"`
	Speed      int    `csv:"Speed"`
	Generation int    `csv:"Generation"`
	Legendary  bool   `csv:"Legendary"`
}
```